engine/src/flutter/impeller/entity/geometry/shadow_path_geometry_unittests.cc

```diff
@@ -0,0 +1,344 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/impeller/entity/geometry/shadow_path_geometry.h"
+
+#include "flutter/display_list/geometry/dl_path.h"
+#include "flutter/display_list/geometry/dl_path_builder.h"
+#include "gtest/gtest.h"
+
+#include "flutter/third_party/skia/src/core/SkVerticesPriv.h"  // nogncheck
+#include "flutter/third_party/skia/src/utils/SkShadowTessellator.h"  // nogncheck
+
+#define SHOW_VERTICES false
+
+namespace impeller {
+namespace testing {
+
+using namespace flutter;
```

Comment:
using namespace is technically forbidden, can you list the symbols you need?

Reply:
Most of the unit test files use the entire flutter namespace.

We even have implementation files that do this as well. :(

Reply:
leave the campsite cleaner that you found it =)